watertap/unit_models/zero_order/tests/test_nanofiltration_zo.py

```diff
@@ -384,20 +384,20 @@ def test_solution(self, model):
             model.fs.unit.properties_treated[0].conc_mass_comp["dye"]
         )
 
-        assert pytest.approx(0.000174141, rel=1e-5) == value(
+        assert pytest.approx(0.00010449, rel=1e-5) == value(
             model.fs.unit.properties_byproduct[0].flow_vol
         )
-        assert pytest.approx(60.794315, rel=1e-5) == value(
+        assert pytest.approx(69.506485, rel=1e-5) == value(
             model.fs.unit.properties_byproduct[0].conc_mass_comp["tds"]
         )
-        assert pytest.approx(14.237023, rel=1e-5) == value(
+        assert pytest.approx(23.710346, rel=1e-5) == value(
```

Comment:
This seems a large difference, was this expected?

Reply:
Yea, I think a large change should be expected since the tds removal frac was changed from 0.212 to 0.145